src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (!newRobot[robot[key]]) {
```

Comment:
If reversed robot will have this property, but it will be falsy - `''`, `0`, `null` or `undefined`, your code will work incorrectly.
Need to use another way to check if object has certain property.